src/controllers/user-controller.js

```diff
@@ -131,7 +174,27 @@ router.get("/prospective/:program", async (req, res) => {
  */
 router.post("/login", async (req, res) => {
     try {
-        
+        initializePassport(
+            passport, 
+            email => Users.findOne({ email: email })
+        )
+        passport.authenticate('local', { session: false }, (err, user, info) => {
+            if (err || !user) {
+                return res.status(400).json({
+                    message: info ? info.message : '로그인에 실패하였습니다.',
+                });
+            }
+    
+            req.login(user, { session: false }, (err) => {
+                if (err) {
+                    res.send(err);
+                }
+    
+                const token = jwt.sign({ id: user.id, email: user.email }, jwtSecret);
```

Comment:
## Possible Issue
Having a JWT that does not expire will cause a problem if it is leaked to a malicious attacker in the future. Therefore, it should have a designated expiration dates such as 24 hours.

Reply:
Modification has been made for this issue. 
The token now expires after 12 hours.